### ISSUE TITLE
Allow to set tab-mute to a specific value

### ISSUE THREAD

**Issue by JarnoB** [qutebrowser/qutebrowser#7456]
Hello,

Currently the only way to mute a specific/current tab you can only toggle the setting with [`tab-mute`](https://qutebrowser.org/doc/help/commands.html#tab-mute). You can of course also set the `content.mute' property to true to do it globally. But where `content.mute` allows to set it to a specific value, `tab-mute` does not.

It would be really useful if a user could set `tab-mute` to a specific value, forcing the tab to be either muted or unmuted. For instance, I would like to open the current (often Youtube) url in mpv, while making sure the tab is muted so the video on the page won't blast through MPV's audio. So my binding could look similar to this:

 `config.bind('<F2>', 'tab-mute  true ;; spawn -d tsp mpv --fullscreen --osd-font-size=20 {url}')`

Additionally, being able to force mute/unmute on a tab would also make it more consistent with the similar `content.mute` option.

Thanks for all your work on Qutebrowser. I've been using it for over a year now and I really enjoy using it!

**Comment by The-Compiler** [comment]
Makes sense! In case you (or someone else) wanted to try opening a PR for this, the relevant code is here:

https://github.com/qutebrowser/qutebrowser/blob/dbcc0e251412330f485dd7f17d67dfa27f4975fd/qutebrowser/components/misccommands.py#L340-L353

**Comment by JarnoB** [comment]
Thanks, that's very helpful! I might give it a try this weekend

**Comment by Dleavenworth** [comment]
Any updates on this? If not, I would like to take this issue on. Thanks!

**Comment by JarnoB** [comment]
I haven't gotten the time to properly look into it yet. Feel free to take it on :)

**Comment by Dleavenworth** [comment]
Great, thanks again! I'll start working on this in the coming week.

**Comment by JarnoB** [comment]
Great! Good luck and have fun :)
